src/azure.ts

```diff
@@ -53,7 +53,7 @@ export const createCluster = (classRef: TerraformStack) => {
     defaultNodePool: {
       name: `defaultpool`,
       vmSize: "Standard_B2s", // 4GB RAM, 8 GB Storage $.0416 / hr
-      nodeCount: 5,
+      nodeCount: 10,
```

Comment:
Curious: Is this so there's room to spare in the cluster for new deployments? Or larger dataset ingestions?

This substantially raises the default cluster cost from .45 an hour to 1.66

Either way, not breaking the bank, but curious what encouraged this change.

Reply:
Been doing some performance testing. Trying to ingest ~ 200MB - 300MB in concurrent ingestions requires larger pods. Seems that memory isn't the primary issue, but rather the CPU, although some pods needed an initial memory boost.

I'm assuming this is because kafka messages are somewhat small, but how fast they can process is the real bottleneck.

Getting a high CPU pool is probably the correct thing to do here, but was just trying to get an answer to if UrbanOS can handle a few hundred MB through configuration.

Reply:
Makes sense, thanks for taking time to explain 👍